content/en/observability_pipelines/installation.md

```diff
@@ -66,10 +66,19 @@ $ DD_API_KEY=<DD_API_KEY> DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.
     sudo apt-get install observability-pipelines-worker datadog-signing-keys
     ```
 
-4. Start the Worker:
+4. Save your Observability Pipelines configuration:
 
     ```
-    DD_API_KEY=<DD_API_KEY> DD_OP_CONFIG_KEY=<DD_OP_CONFIG_KEY> observability-pipelines-worker 
+    echo "<DD_OP_CONFIG>" > /var/lib/observability-pipelines-worker/observability-pipelines-worker.yaml
+    ```
+
+    Where `DD_OP_CONFIG` are the contents of your Observability Pipeline configuration that you created in the [Observability Pipelines UI].
```

Comment:
:two_cents:
```suggestion
    Where `DD_OP_CONFIG` is the contents of your Observability Pipeline configuration that you created in the [Observability Pipelines UI].
```

Reply:
Also does `[Observability Pipelines UI]` need a link?

Reply:
+1 on both comments. Also can we update `contents` to `content`?

Reply:
Thanks. Done.